src/app/actions/newsDownloaded.js

```diff
@@ -0,0 +1,10 @@
+import constants from '../constants';
+
+function newsDownloaded(newArticles) {
```

Comment:
I think it's much easier to navigate through action when you have action creators and functions for side effects (AJAX) in the same file.
Also, please take a look on redux-actions library, it'll help you make action creators in one line